pkg/providers/aws/provider_postgres.go

```diff
@@ -806,11 +808,11 @@ func (p *PostgresProvider) exposePostgresMetrics(ctx context.Context, cr *v1alph
 	resources.SetMetricCurrentTime(resources.DefaultPostgresInfoMetricName, infoLabels)
 
 	// set available metric
-	if instance == nil || *instance.DBInstanceStatus != "available" {
+	if len(string(cr.Status.Phase)) == 0 || cr.Status.Phase != croType.PhaseComplete {
 		resources.SetMetric(resources.DefaultPostgresAvailMetricName, genericLabels, 0)
 		return
 	}
-	resources.SetMetric(resources.DefaultPostgresAvailMetricName, genericLabels, 1)
+	resources.SetMetric(resources.DefaultPostgresAvailMetricName, infoLabels, 1)
```

Comment:
I don't think this should change, the availablity metric should be left as is with its generic labels

Reply:
maybe ? Only really getting a good look a the metrics now that I have them applied in the monitoring stack.

Reply:
your right.